src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/IO/Packaging/indexingfiltermarshaler.cs

```diff
@@ -215,8 +215,6 @@ internal static IntPtr MarshalPropVariant(Object obj)
 
                 // allocate an unmanaged PROPVARIANT to return
                 pNative = Marshal.AllocCoTaskMem(Marshal.SizeOf(typeof(PROPVARIANT)));
-                // Per MSDN, AllocCoTaskMem never returns null. One can't be too careful, though.
-                Invariant.Assert(pNative != null);
```

Comment:
Better to assert  pNative != IntPtr.Zero  to match the intent of the comment

Reply:
Changed here: https://github.com/dotnet/wpf/pull/3649/commits/fc1773490aba4d14338c3caa38e3ca210e7a958d#diff-a630013d4fe19d9fdfba7cdfbdb106e1426139fc68bd312431a2cc9a3a9f6c02R220